systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -979,6 +979,13 @@ public static void createFileSinkConnector(String podName, String topicName) {
         );
     }
 
+    public static void waitUntilKafkaConnectServiceIsPresent(String podNamePrefix) {
```

Comment:
I wonder if we should rename this. From this name I expected it to be checking the Kubernetes service. MAybe calling is `waitUntilKafkaConnectRestApiIsAvailable` would work better.

Reply:
make sense